presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -447,6 +447,83 @@ Property Name                                        Description
                                                      defaults to ``20``.
 ==================================================== ============================================================
 
+Performance Tuning Configuration Properties
+-------------------------------------------
+
+The following table describes performance tuning properties for the Hive
+connector.
+
+.. warning::
+
+   Performance tuning configuration properties are considered expert-level
+   features. Altering these properties from their default values is likely to
+   cause instability and performance degradation. We strongly suggest that
+   you use them only to address non-trivial performance issues, and that you
+   keep a backup of the original values, if you change them. 
+
+.. list-table:: Hive performance tuning configuration properties
+    :widths: 30, 50, 20
+    :header-rows: 1
+
+    * - Property name
+      - Description
+      - Default value
+    * - ``hive.domain-compaction-threshold``
+      - Sets the number of transactions to act as threshold. Once reached the
```

Comment:
This description is not correct at all. Let's remove it for now, since it would take far too much effort to try and explain what it does and the implications of changing it.